src/main/java/frc/robot/commands/rests/restUtils/RESTHandler.java

```diff
@@ -0,0 +1,272 @@
+package frc.robot.commands.rests.restUtils;
+
+import edu.wpi.first.util.datalog.DataLog;
+import edu.wpi.first.util.datalog.StringLogEntry;
+import edu.wpi.first.util.sendable.Sendable;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.util.sendable.SendableRegistry;
+import edu.wpi.first.wpilibj.DataLogManager;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import frc.robot.commands.rests.RESTCommand;
+import frc.robot.commands.rests.restAnnotations.*;
+
+import javax.inject.Inject;
+import java.lang.annotation.AnnotationTypeMismatchException;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Objects;
+
+public class RESTHandler implements Sendable, AutoCloseable {
```

Comment:
This class is great -- well written and should help make it easy for reuse by others later! With that in mind, it'd be good to add JavaDoc annotations so those using it later would have clear documentation in the code to reference.